adapters/MoPub/MoPubAdapter/MoPubAdapterMediatedNativeAd.m

```diff
@@ -48,12 +48,16 @@ - (instancetype)initWithMoPubNativeAd:(nonnull MPNativeAd *)moPubNativeAd
     }
     else {
       NSURL *mainImageUrl =
-      [[NSURL alloc] initFileURLWithPath:[_nativeAdProperties objectForKey:kAdMainImageKey]];
+      [[NSURL alloc] URLWithString:[_nativeAdProperties objectForKey:kAdMainImageKey]];
```

Comment:
I think this needs to be `NSURL *mainImageUrl = [NSURL URLWithString:[_nativeAdProperties objectForKey:kAdMainImageKey]];`

Reply:
Thanks! It's updated.